roles/security_errata/tasks/main.yml

```diff
@@ -1,11 +1,13 @@
 ---
-- name: Patch CVE-2014-0160
+- name: update apt index
+  apt: update_cache=yes cache_valid_time=3600
+
+- name: update SSL
   apt: pkg={{ item }} state=latest
   with_items:
   - libssl1.0.0
   - openssl
-  notify: reload haproxy
-  tags: CVE-2014-0160
+  notify: restart haproxy
 
 - include: CVE-2015-0235.yml
-  tags: CVE-2015-0235
\ No newline at end of file
+  tags: CVE-2015-0235
```

Comment:
nah, there's like 12 CVEs for it.   I think just the security tag should be enough.